.drone.yml

```diff
@@ -195,25 +195,6 @@ steps:
     event:
     - push
 
-- name: bin-publish
-  image: plugins/gcs
-  settings:
-    acl:
-    - allUsers:READER
-    cache_control: "public,no-cache,proxy-revalidate"
-    source: bin/rancherd
-    target: releases.rancher.com/rancherd/${DRONE_BRANCH}/rancherd
-    token:
-      from_secret: google_auth_key
-  when:
-    ref:
-      include:
-      - "refs/heads/master"
-      - "refs/heads/release/v*"
-    event:
-    - push
-
-
```

Comment:
This one is debatable, since all we really need to do is remove it from the release. But I think for now removing it is the right thing, so if people are using automation to go find it then they can't anymore.